graph-image/src/lib/GraphImage/GraphImage.svelte

```diff
@@ -5,7 +5,6 @@
 
 	export let image: GraphAsset;
 	export let maxWidth: number = 800;
-	export let fadeIn: boolean = true;
```

Comment:
Some users may want to disable fading entirely, we should still allow this option